src/luascript.cpp

```diff
@@ -1895,6 +1895,7 @@ void LuaScriptInterface::registerFunctions()
 	registerEnumIn("configKeys", ConfigManager::REMOVE_WEAPON_AMMO)
 	registerEnumIn("configKeys", ConfigManager::REMOVE_WEAPON_CHARGES)
 	registerEnumIn("configKeys", ConfigManager::REMOVE_POTION_CHARGES)
+    registerEnumIn("configKeys", ConfigManager::PZLOCK_SKULL_ATTACKER)
```

Comment:
```suggestion
	registerEnumIn("configKeys", ConfigManager::PZLOCK_SKULL_ATTACKER)
```

Reply:
that's weird i don't know why it end up that way